src/main/java/by/dragonsurvivalteam/dragonsurvival/magic/abilities/ForestDragon/active/SpikeAbility.java

```diff
@@ -148,13 +155,25 @@ public void onCast(Player player){
 
 		DragonStateHandler handler = DragonUtils.getHandler(player);
 		handler.getMovementData().bite = true;
+		
+		for (int i = 0; i < getLevel(); i++) {
+			DragonSpikeEntity entity = new DragonSpikeEntity(DSEntities.DRAGON_SPIKE, player.level, player);
+			entity.setPos(entity.getX() + d2, entity.getY() + d3, entity.getZ() + d4);
+			entity.setArrow_level(getLevel());
+			entity.setBaseDamage(getDamage());
+			entity.pickup = AbstractArrow.Pickup.DISALLOWED;
+			entity.shootFromRotation(player, player.xRot, player.yRot, 0.0F, 4F, i * spikeSpread);
+			player.level.addFreshEntity(entity);
+			if (!spikeMultishot)
+				break;
+		}
 
-		DragonSpikeEntity entity = new DragonSpikeEntity(DSEntities.DRAGON_SPIKE, player.level, player);
-		entity.setPos(entity.getX() + d2, entity.getY() + d3, entity.getZ() + d4);
-		entity.setArrow_level(getLevel());
-		entity.setBaseDamage(getDamage());
-		entity.pickup = AbstractArrow.Pickup.DISALLOWED;
-		entity.shootFromRotation(player, player.xRot, player.yRot, 0.0F, 4F, 1.0F);
-		player.level.addFreshEntity(entity);
+		//DragonSpikeEntity entity = new DragonSpikeEntity(DSEntities.DRAGON_SPIKE, player.level, player);
+		//entity.setPos(entity.getX() + d2, entity.getY() + d3, entity.getZ() + d4);
+		//entity.setArrow_level(getLevel());
+		//entity.setBaseDamage(getDamage());
+		//entity.pickup = AbstractArrow.Pickup.DISALLOWED;
+		//entity.shootFromRotation(player, player.xRot, player.yRot, 0.0F, 4F, 1.0F);
+		//player.level.addFreshEntity(entity);
```

Comment:
just remove code thats no longer needed, git history is enough

Reply:
It's gone now.